src/ServicePulse.Host/app/modules/configuration/views/notificationsemailmodal.html

```diff
@@ -0,0 +1,56 @@
+﻿<div class="modal-header">
+    <h3 class="modal-title">Email configuration</h3>
+</div>
+<form name="notificationsForm" novalidate>
+    <div class="modal-body">
+        <div class="row">
+            <div class="form-group" ng-class="{'has-error': notificationsForm.smtpServerAddress.$invalid}">
+                <label for="smtpServerAddress">SMTP server address</label>
+                <input type="text" id="smtpServerAddress" name="smtpServerAddress"
+                    ng-model="settings.smtp_server"
+                    class="form-control" required />
+            </div>
+            <div class="row"></div>
+            <div class="form-group" ng-class="{'has-error': notificationsForm.smtpServerPort.$invalid}">
+                <label for="smtpServerPort">SMTP server port</label>
+                <input type="text" id="smtpServerPort" name="smtpServerPort"
+                    ng-model="settings.smtp_port"
+                    class="form-control" required />
+            </div>
+            <div class="row"></div>
+            <div class="form-group">
+                <label for="account">Authentication account</label>
+                <input type="text" id="account" name="account"
+                    ng-model="settings.authentication_account" class="form-control" />
+            </div>
+            <div class="row"></div>
+            <div class="form-group">
+                <label for="account">Authentication password</label>
+                <input type="password" id="password" name="password"
+                    ng-model="settings.authentication_password" class="form-control" />
+            </div>
+            <div class="row"></div>
+            <div class="form-group">
+                <input type="checkbox" id="enableSSL" name="enableSSL"
```

Comment:
As far as I recall TLS is superseeded SSL and email also has STARTTLS. Should this be renamed to something like EnableEncryption?

In outlook this is called Encryption method

![image](https://user-images.githubusercontent.com/174258/115863928-769b2380-a436-11eb-9324-5ae601adcc2f.png)

Reply:
I've changed the label (and all types involved) to `EnableTLS`